content/docs/reference/tracing.mdx

```diff
@@ -16,109 +16,136 @@ import TabItem from '@theme/TabItem';
 
 ## Summary
 
-Tracing tracks the progression of a single user request as it is handled by Pomerium.
+Pomerium has comprehensive support for OpenTelemetry tracing, allowing detailed introspection into requests and authorization flows. You can use tracing to debug errors and latency issues in your applications.
```

Comment:
The first time we say OpenTelemetry, let's say "OpenTelemetry (OTel)" so later the OTel reference is clear.